### PR TITLE
Adds engineering imprinter

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Circuit imprinters in engineering will be "engineering circuit imprinter" instead of unregistered one.
## Changelog
:cl:
add: Special circuit imprinter subtype for engineering
/:cl:

### CODE REVIEW

File: code/modules/research/machinery/departmental_circuit_imprinter.dm [hunk 1]
```diff
@@ -3,10 +3,15 @@
...
 
 /obj/machinery/rnd/production/circuit_imprinter/department/science
 	name = "department circuit imprinter (Science)"
 	circuit = /obj/item/circuitboard/machine/circuit_imprinter/department/science
 	allowed_department_flags = DEPARTMENTAL_FLAG_ALL|DEPARTMENTAL_FLAG_SCIENCE
 	department_tag = "Science"
+
+/obj/machinery/rnd/production/circuit_imprinter/department/engineering
+	name = "department circuit imprinter (Science)"
+	circuit = /obj/item/circuitboard/machine/circuit_imprinter/department/science
```
Comment (KubeRoot) [inline code/modules/research/machinery/departmental_circuit_imprinter.dm]: This should have a different circuit

**Review by KubeRoot — CHANGES_REQUESTED** [review]

File: code/modules/research/machinery/departmental_circuit_imprinter.dm (hunk 1 above)
Comment (KubeRoot) [reply code/modules/research/machinery/departmental_circuit_imprinter.dm]: In fact, the name is also wrong? Not sure if it's actually used in-game, but it should be changed at least for the editor.

And are you sure about the department flags?

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 2]
```diff
@@ -804,6 +804,11 @@
 	icon_state = "science"
 	build_path = /obj/machinery/rnd/production/circuit_imprinter/department/science
 
+/obj/item/circuitboard/machine/circuit_imprinter/department/engineering
+	name = "departmental circuit imprinter - engineering (Machine Board)"
+	icon_state = "science"
```
Comment (KubeRoot) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]:
```suggestion
	icon_state = "engineering"
```

**Comment by BrumalLake** [comment]
this pr just expired with https://github.com/BeeStation/BeeStation-Hornet/pull/5675
other than the engineering printer part

**Comment by KubeRoot** [comment]
I would recommend renaming the PR and updating the description to match the current state, since it no longer affects the ability to access circuit imprinters and only adds the engineering circuit imprinter.

**Review by francinum — CHANGES_REQUESTED** [review]
Engineering is not getting a free autolathe.

**Comment by francinum** [comment]
PR body has insufficient information and insufficient justification for alteration.

**Comment by Archanial** [comment]
PR body has been updated

**Review by francinum — APPROVED** [review]

**Pull request merged**